exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -264,19 +265,26 @@ private ServerConnector createHttpsConnector() throws Exception {
 
     final SslContextFactory sslContextFactory = new SslContextFactory();
 
-    if (config.hasPath(ExecConstants.HTTP_KEYSTORE_PATH) &&
-        !Strings.isNullOrEmpty(config.getString(ExecConstants.HTTP_KEYSTORE_PATH))) {
-      logger.info("Using configured SSL settings for web server");
-      sslContextFactory.setKeyStorePath(config.getString(ExecConstants.HTTP_KEYSTORE_PATH));
-      sslContextFactory.setKeyStorePassword(config.getString(ExecConstants.HTTP_KEYSTORE_PASSWORD));
-
-      // TrustStore and TrustStore password are optional
-      if (config.hasPath(ExecConstants.HTTP_TRUSTSTORE_PATH)) {
-        sslContextFactory.setTrustStorePath(config.getString(ExecConstants.HTTP_TRUSTSTORE_PATH));
-        if (config.hasPath(ExecConstants.HTTP_TRUSTSTORE_PASSWORD)) {
-          sslContextFactory.setTrustStorePassword(config.getString(ExecConstants.HTTP_TRUSTSTORE_PASSWORD));
-        }
+    final boolean hasPath = config.hasPath(ExecConstants.HTTP_KEYSTORE_PATH);
```

Comment:
To avoid this double-checking, standard practice is:

* Use a blank value to indicate the value is unset.
* Provide a default (blank) value in `drill-module.conf`

Then, the code can just be:

```
String path = config.getString(ExecConstants.HTTP_KEYSTORE_PATH).trim();
if (! path.isEmpty()) {
  // do stuff
```

Also, if the user is expected to set this key, please provide an example in `drill-override-example.conf`.